content/en/blog/_posts/2021-09-21-clusterclass-and-managed-topologies.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: blog
+title: "Introducing ClusterClass and Managed Topologies in Cluster API"
+date: 2021-09-21
+slug: capi-clusterclass-and-managed-topologies
+---
+
+**Author:** Fabrizio Pandini (VMware)
+
+The [Cluster API community](https://cluster-api.sigs.k8s.io/) is happy to announce the implementation of *ClusterClass and Managed Topologies*, a new feature that will greatly simplify how you can provision, upgrade, and operate multiple Kubernetes clusters in a declarative way.
+
+## A little bit of context…
+
+Before getting into the details, let's take a step back and look at the history of Cluster API.
+
+The [Cluster API project](https://github.com/kubernetes-sigs/cluster-api/) started three years ago, and the first releases focused on extensibility and implementing a declarative API that allows a seamless experience across infrastructure providers. This was a success with many cloud providers: AWS, Azure, vSphere, Metal3, Digital Ocean, GCP and still counting.
```

Comment:
nit: CNCF policy for listing 3rd party products and / or vendors is to use either alphabetical order, or a random shuffle.
How about:
```suggestion
The [Cluster API project](https://github.com/kubernetes-sigs/cluster-api/) started three years ago, and the first releases focused on extensibility and implementing a declarative API that allows a seamless experience across infrastructure providers. This was a success with many cloud providers: AWS, Azure, Digital Ocean, GCP, Metal3, vSphere…  and still counting.
```

Reply:
@sftim Out of curiosity: how do you now something uses "random shuffle"? :)